Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -59,11 +60,201 @@
 #import <WebCore/ResourceResponse.h>
 #import <WebCore/ScrollAnimator.h>
 #import <WebCore/SharedBuffer.h>
+#import <WebCore/WebAccessibilityObjectWrapperMac.h>
 #import <wtf/cocoa/VectorCocoa.h>
 #import <wtf/text/TextStream.h>
-
```

Comment:
This newline was here on purpose (softlink headers have to be in their own paragraph at the end)